app/views.py

```diff
@@ -41,6 +41,48 @@ def upload():
     return flask.render_template('upload.html', form = form)
 
 
+def positionsPrinter(length):
+    """
+    Create a string containing the position numbers in a way that they will be
+    aligned with the sequence.
+    ARGUMENT:
+        length of the sequence to consider.
+    """
+    numbers = range(0, 10000, 10)
+    numbers[0] = 1
+    pos = ""
+    for i in xrange((length/10)+1):
+        if i == 0:
+            pos += "{:<9d}".format(numbers[0])
+        else:
+            pos += "{:<10d}".format(numbers[i])
```

Comment:
en Python 3, je crois qu'on peut pas acceder à un element comment ca. 
Mais du coup, en utilisant number dans le for, ya pas besoin de l'indexation.